packages/node/src/utils/commonjs.ts

```diff
@@ -1,4 +1,4 @@
 /** Detect CommonJS. */
 export function isCjs(): boolean {
-  return typeof require !== 'undefined';
+  return typeof module !== 'undefined' && typeof module.exports !== 'undefined';
```

Comment:
Why are we also checking for module.exports?

Reply:
Just some additional safety as there is maybe a variable named `module`. But it's more unlikely that there is one that is an object with the key `exports`.